internal/service/connect/quick_connect.go

```diff
@@ -0,0 +1,420 @@
+package connect
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/connect"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
+)
+
+func ResourceQuickConnect() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceQuickConnectCreate,
+		ReadContext:   resourceQuickConnectRead,
+		UpdateContext: resourceQuickConnectUpdate,
+		DeleteContext: resourceQuickConnectDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(connectQuickConnectCreatedTimeout),
```

Comment:
You provide Timeouts, and the ability to override via Configuration, but do not have corresponding wait() and status() functions in the resource implementation. After reviewing the Connect service, I see this is actually pretty common, meaning that even though we have Timeout values exposed to users, we don't actually honor them.

The question becomes, do we actually need to provide timeouts? These are typically reserved for resources that have eventual consistency/asynchronous APIs.

See a good example of honoring timeouts here: https://github.com/hashicorp/terraform-provider-aws/blob/f445f5e1980a8b4a6581b2964b8ba70b80330128/internal/service/lexmodels/bot.go#L268

Reply:
Resolved in bc0343b81d70556e78c6399bf73e75b1751cf96f by removing timeouts

Reply:
Missed some unused variables in the earlier commit. Removed them here: c2c12e6a176c8304af7c3261c5c9932868ca6186